public/css/lrr.css

```diff
@@ -771,6 +782,25 @@ div.fullscreen img {
   overflow-y: scroll;
 }
 
+.thumbnail-bookmark-icon {
+  cursor: pointer;
+  position: absolute;
+  left: 10px;
+  bottom: 20px;
+  z-index: 2;
+  font-size: 24px;
+  color: white;
```

Comment:
```suggestion
```
I'd remove this attribute outright so the bookmark button uses the text color for the current theme instead.  

![image](https://github.com/user-attachments/assets/56c83f5d-71ac-4e50-9579-a7f4efe5557f)
![image](https://github.com/user-attachments/assets/c89d028d-ce7a-42f8-bff2-defb3687a458)
![image](https://github.com/user-attachments/assets/fec7873e-9d26-4287-9c94-c4184937b86a)

(Using white has a bit of a readability issue against B/W manga pages, but the text shadow is enough to handle that IMO)